javascript/javascript2/week3/game/game.js

```diff
@@ -0,0 +1,79 @@
+const btnTag = document.getElementById("start-btn");
```

Comment:
check this if this can reduce code.

'const _= e => document.getElementById(e);

const wrapper = _('wrapper'),
    CovidData = _('CovidData'),
    summary =   _('summary'),